oauthenticated.cabal

```diff
@@ -1,87 +1,118 @@
-name:                oauthenticated
-version:             0.1.3.4
-synopsis:            Simple OAuth for http-client
+-- This file has been generated from package.yaml by hpack version 0.20.0.
```

Comment:
Probably just delete this file from the repo since it's now generated at build time?

Reply:
I think some people building with cabal will still need it?